automathor.py

```
@@ -0,0 +1,170 @@
+    file: str
+    line: int
+    text: str
+
+    def get_git_blame_output(self) -> Context:
+        self.line = int(self.line)
+
+        start_line = max(1, self.line - 4)
+        end_line = self.line + 4
+        command = f"git blame --show-email {self.file} | sed -n '{start_line},{end_line}p'"
+        logging.debug(f"RUNNING {command}")
+        result = subprocess.run(command, shell=True, capture_output=True, text=True)
```

Ruff (0.8.2)

`subprocess` call with `shell=True` identified, security issue

(S602)